docs/VnVPlan/VnVPlan.tex

```diff
@@ -188,50 +189,138 @@ \subsection{Tests for Functional Requirements}
 \wss{Include a blurb here to explain why the subsections below
 	cover the requirements.  References to the SRS would be good here.}
 
-\subsubsection{Area of Testing1}
-
-\wss{It would be nice to have a blurb here to explain why the subsections below
-	cover the requirements.  References to the SRS would be good here.  If a section
-	covers tests for input constraints, you should reference the data constraints
-	table in the SRS.}
+\subsubsection{Work Orders}
 
-\paragraph{Title for Test}
+This section will contains tests covering the ``Work Orders'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
 
 \begin{enumerate}
 
-	\item{test-id1\\}
+	\item \textbf{FRT-BE17-1}
 
-	Control: Manual versus Automatic
+	      Control: Manual
 
-	Initial State:
+	      Initial State: Shop Owner/Employee Account with a registered shop and a quote
 
-	Input:
+	      Input: A new appointment created at the shop with a quote ID
```

Comment:
Is the ID an input if it is automatically generated by the system? I think we said the system would do this right?

Reply:
I think what I was trying to say is that the appointment is linked to the quote.